integration-test/src/java/com/twitter/heron/integration_test/topology/serialization/CustomObject.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.integration_test.topology.serialization;
+
+import java.io.Serializable;
+
+/**
+ * An example Java-serializable custom object.
+ */
+public class CustomObject implements Serializable {
+
+  private static final long serialVersionUID = -7502839214902002412L;
+  private String name;
+  private Integer age;
+
+  public CustomObject(String name, Integer age) {
```

Comment:
@billonahill Yes, I will add other types, also make sure serialization/deserialization works fine in the intermediate bolts too.